### PR TITLE
cnao, Add stable lane release-0.58

### PR DESCRIPTION
**Pull request by RamLavi**
This PR adds a stable CNAO lane to test PRs on release-0.58 branch

### CODE REVIEW

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

File: github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.58.yaml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      branches:
+        - release-0.58
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 4h
+        grace_period: 10m
+      cluster: prow-workloads
+      max_concurrency: 6
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.58.yaml:14]: maybe worth to reduce to 2 / 3
same all over

**Comment by RamLavi** [comment]
fixed [here](https://github.com/kubevirt/project-infra/compare/e9850ee6f3cf86a945f5ea79178b5a2d769097c5..668a73ba2ad534c0c2e0753959ce7ef330b01c09)

File: github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.58.yaml [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+      cluster: prow-workloads
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.58.yaml:206]: not saying that it should be changed in this PR - just note this image is different from the others

**Review by oshoval — APPROVED** [review]

**Comment by RamLavi** [comment]
@phoracek can you re-lgtm+approve here as well?

**Pull request merged**